src/pytom_tm/entry_points.py

```diff
@@ -555,12 +555,25 @@ def match_template(argv=None):
              "computations for non-spherical masks and roughly doubles computation time.",
     )
     rotation_group = parser.add_argument_group('Angular search')
+    rotation_group.add_argument(
+        "--particle-diameter",
+        type=float,
+        required=False,
+        action=LargerThanZero,
+        help="Provide a particle diameter (in Angstrom) to automatically determine the "
+             "angular sampling using the Crowther criterion. For the max resolution "
+             "the pixel size is used unless a low-pass filter is specified, in which "
```

Comment:
it is pixelsize * 2, right?
```suggestion
             "the 2 * pixel size is used unless a low-pass filter is specified, in which "
```